### ISSUE TITLE
Test 4.1.1-A assumes a version on demand system

### ISSUE THREAD

**Issue by whikloj** [fcrepo/Fedora-API-Test-Suite#338]
Test fails if there is a version returned, which is fine when Fedora 5 (manual versioning) was how the test was confirmed.

https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/versioning/LdprvHttpGet.java#L72

**Comment by awoods** [comment]
In order for F6 to pass this test, we need to set auto-versioning to `false`?

**Comment by whikloj** [comment]
Yes it verifies it is a versionable resource with a TimeGate, but then tries to get a version and assumes there won't be one. Because F6 is OCFL and auto-versioning it always has the first version when created.

**Comment by awoods** [comment]
I wonder what the broader impact would be if we run the tests with auto-versioning set to `false`.
